src/txn/txn_ckpt.c

```diff
@@ -352,11 +352,8 @@ __checkpoint_reduce_dirty_cache(WT_SESSION_IMPL *session)
     conn = S2C(session);
     cache = conn->cache;
 
-    /*
-     * Give up if scrubbing is disabled, including when checkpointing with a timestamp on close (we
-     * can't evict dirty pages in that case, so scrubbing cannot help).
-     */
-    if (F_ISSET(conn, WT_CONN_CLOSING_TIMESTAMP) || cache->eviction_checkpoint_target < DBL_EPSILON)
+    /* Give up if scrubbing is disabled. */
```

Comment:
With WT-8379, eviction is allowed in parallel to the shutdown checkpoint. The comment and code don't valid now, so I removed them to allow scrubbing in the shutdown checkpoint. Let me know if I can move these changes into a separate ticket?